<launch>
	<!-- Startup the camera as a robot listening to commands and delivering images -->
	<!-- node name="cam_actuator_node" pkg="logitech_cam" type="cam_actuator_node" / -->
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    	<param name="image_width" value="640" />
    	<param name="image_height" value="480" />
		<param name="pixel_format" value="yuyv"/>
	</node>
	<node name="camera_reader" pkg="logitech_cam" type="camera_reader.py" output="screen"/>
	<node name="image_view" pkg="image_view" type="image_view">
		<remap from="image" to="/usb_cam/image_raw" />
	</node> 
	<node name="PIL_Image" pkg="image_view" type="image_view">
		<remap from="image" to="/camera_reader/image" />
	</node> 
</launch>